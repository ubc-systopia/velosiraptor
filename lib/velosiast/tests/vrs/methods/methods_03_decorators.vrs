
// method with post-condition
#[remap]
abstract fn foo() -> bool
    ensures true;
