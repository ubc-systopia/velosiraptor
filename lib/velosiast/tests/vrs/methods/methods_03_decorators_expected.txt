  #[remap]
  abstract fn foo() -> bool
    requires true;
  { }

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@